src/test/java/org/folio/circulation/domain/UpdateRequestQueueTest.java

```diff
@@ -0,0 +1,105 @@
+package org.folio.circulation.domain;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.folio.circulation.domain.reorder.ReorderQueueRequest;
+import org.folio.circulation.domain.reorder.ReorderRequest;
+import org.folio.circulation.resources.context.ReorderRequestContext;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.CollectionResourceClient;
+import org.folio.circulation.support.ForwardOnFailure;
+import org.folio.circulation.support.Result;
+import org.folio.circulation.support.http.client.Response;
+import org.junit.Test;
+
+import api.support.builders.RequestBuilder;
+import io.vertx.core.json.JsonObject;
+
+public class UpdateRequestQueueTest {
```

Comment:
```suggestion
public class ReorderRequestQueueTest {
```

Reply:
This is a unit test, so I named it as the target class.

Reply:
That is likely my style, I tend to name my test classes after the context / scenarios that the class covers rather than specifically the class(es) that are involved.